Aquality.Selenium.Template/Aquality.Selenium.Template/Elements/CustomElements.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Aquality.Selenium.Core.Elements;
+using Aquality.Selenium.Elements;
+using OpenQA.Selenium;
+
+namespace Aquality.Selenium.Template.Elements
+{
+    public class CustomElemвents
+    {
+        public class CustomTextBox : TextBox
```

Comment:
nested public classes is a bad code style I believe. Could you please run your code through the sonar quality gate?

Reply:
Fixed it. Just using the CustomTextBox class.